spec/requests/api/user_can_view_profile_spec.rb

```diff
@@ -0,0 +1,43 @@
+RSpec.describe 'GET /api/users/:user_id', type: :request do
+  let(:user) { create(:user, email: 'test-2@example.com') }
+  let(:user_header) { user.create_new_auth_token }
+  let!(:post) { create(:post, user_id: user.id) }
+
+  describe 'successfully' do
+    before do
+      get "/api/users/#{user.id}",
+        headers: user_header
+    end
+
+    it 'is expected to return a 200 status' do
+      expect(response).to have_http_status 200
+    end
+
+    it 'is expected to return a user email' do
+      expect(response_json['user']['email']).to eq 'test-2@example.com'
+    end
+
+    it 'is expected to return the track of the post' do
+      expect(response_json['user']['posts'][0]['track']).to eq 'MyTrack'
+    end
+
+    it 'is expected to return the descrption of the post' do
```

Comment:
You have a small typo here 😄. You forgot `i` in description

Reply:
Fixed now 👍